Sources/S3DocArchives/S3DocArchives.swift

```diff
@@ -96,6 +145,16 @@ private extension S3 {
             return eventLoop.makeSucceededFuture((true, accumulator + files))
         }
     }
+
+    func getDocArchiveTitle(in bucket: String,
+                            path: S3DocArchives.DocArchive.Path) async throws -> String? {
+        let key = S3.StoreKey(bucket: bucket,
+                              path: path.s3path + "/data/documentation/\(path.product).json")
```

Comment:
This is the key lookup. After we merge this I'll probably make this package dependent the server project so we don't copy around this file any longer. It's getting too complex for that.